auth/src/desktop/auth_desktop.cc

```diff
@@ -94,9 +94,7 @@ void* CreatePlatformAuth(App* const app) {
 void InitializeFunctionRegistryListener(AuthData* auth_data);
 void DestroyFunctionRegistryListener(AuthData* auth_data);
 
-// TODO(b/211006737): This is a stub until desktop implementation supports
-// heartbeat logging.
-void LogHeartbeat(Auth* const auth) {}
+void LogHeartbeat(Auth* const auth) { auth->auth_data->app->LogHeartbeat(); }
```

Comment:
Though it is very unlikely to have nullptr but please do null check on `auth`, `auth->auth_data` and `auth->auth_data->app` just in case

Reply:
Done. Added a series of conditions within an if